app/services/player.js

```diff
@@ -101,7 +101,8 @@ export default Service.extend({
 		get(this, 'store').findRecord('track', event.track.id)
 			.then(track => {
 				// only set media as not-available for track owner (for now)
-				if(channel.get('id') === track.get('channel.id')) {
+				const isOwner = channel.get('id') === track.get('channel.id')
```

Comment:
as it is only used once, not sure we need to declare a variable here, but that is a detail and I like the clean look of this